content/en/case-studies/deeplabcut-dnn.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Case Study: DeepLabCut 3D Pose Estimation"
+sidebar: false
+---
+
+{{< figure src="/images/content_images/cs/mice-hand.gif" class="fig-center" caption="**Analyzing mice hand-movement using DeepLapCut**" alt="micehandanim" attr="*(Source: www.deeplabcut.org )*" attrlink="http://www.mousemotorlab.org/deeplabcut">}}
+
+<blockquote cite="https://news.harvard.edu/gazette/story/newsplus/harvard-researchers-awarded-czi-open-source-award/">
+    <p>Open Source Software is  accelerating Biomedicine. DeepLabCut enables automated video analysis of animal behavior using Deep Learning.</p>
+    <footer align="right">—Alexander Mathis, <cite>PostDoctoral Fellow, Harvard Chan Zuckerberg Initiative (CZI)</cite></footer>
```

Comment:
I think there's no such thing as `Harvard Chan Zuckerberg Initiative`, and this should just be `Harvard`. CZI just funded one grant, but that doesn't make it an affiliation.